Sources/PipifyViewModifier.swift

```diff
@@ -40,6 +40,9 @@ internal struct PipifyModifier<PipView: View>: ViewModifier {
                     controller.stop()
                 }
             }
+            .onChange(of: controller.enabled) { newValue in
+                isPresented = newValue
+            }
```

Comment:
Do we still need `isPresented` with this pattern?
Could we move the logic from the `.onChange(of: isPresented)` above to this closure?

Reply:
Yes, this pattern is still required. If the user clicks the x button on the pip or if pip closes due to an error, the `isPresented` and `enabled` flags should be synced. This is required to prevent possible state problems.